.github/workflows/rtw-env/rtw-env.yml

```diff
@@ -0,0 +1,11 @@
+name: recipe_test_workflow
+
+channels:
+  - conda-forge
+
+dependencies:
+  - cylc-flow>=8.2
```

Comment:
```suggestion
  - cylc-flow >=8.2
```

Reply:
Added in commit https://github.com/ESMValGroup/ESMValTool/commit/41b0abb8b4096738d1b3a4b803fd933b1c0e5da1. Cheers :slightly_smiling_face: